spec/views/integration_spec.rb

```diff
@@ -0,0 +1,231 @@
+require 'rails_helper'
+RSpec.describe '/users/', type: :system do
+  before :all do
+    User.destroy_all
+    @user_one = User.create(name: 'Jalhandar-1', photo: 'image-1', bio: 'Mahadev Shivansh Jalhander-1 Mahadev')
+    @user_two = User.create(name: 'Jalhandar-2', photo: 'image-1', bio: 'Mahadev Shivansh Jalhander-2 Mahadev')
+    @user_three = User.create(name: 'Jalhandar-3', photo: 'image-1', bio: 'Mahadev Shivansh Jalhander-3 Mahadev')
+    @user_four = User.create(name: 'Jalhandar-4', photo: 'image-1', bio: 'Mahadev Shivansh Jalhander-4 Mahadev')
+    @post_one = Post.create(title: 'Dev Asur-1 youdh', text: 'Shivansh ka Krodh', author: @user_one)
+    @post_three = Post.create(title: 'Asur youdh-1', text: '# Asur krodh me Agni', author: @user_one)
+    @post_two = Post.create(title: 'Dev Asur-2', text: '# Shivansh ne indra ko', author: @user_one)
+    @post_four = Post.create(title: 'Asur youdh-2', text: '# Asur krodh me Agni', author: @user_one)
+    @post_five = Post.create(title: 'Asur youdh-3', text: '# Asur krodh me Agni', author: @user_one)
+    @comment_one = Comment.create(author: @user_one, post: @post_one, text: 'anything u want')
+    @comment_two = Comment.create(author: @user_two, post: @post_one, text: 'anything u want 1')
+    @comment_three = Comment.create(author: @user_three, post: @post_one, text: 'anything u want 2')
+    @comment_four = Comment.create(author: @user_four, post: @post_one, text: 'anything u want 3')
+    @comment_five = Comment.create(author: @user_four, post: @post_one, text: 'anything u want 4')
+    @comment_six = Comment.create(author: @user_four, post: @post_one, text: 'anything u want 4')
+    @like_one = Like.create(author: @user_one, post: @post_one)
+    @like_two = Like.create(author: @user_two, post: @post_one)
+    @like_three = Like.create(author: @user_three, post: @post_one)
+    @like_four = Like.create(author: @user_one, post: @post_one)
+    @like_five = Like.create(author: @user_four, post: @post_one)
+    @like_six = Like.create(author: @user_one, post: @post_one)
+    @users = User.all
```

Comment:
## [OPTIONAL]

- [ ] It would be better to structure your integration test per path/feature so that people can test a given single feature(login, create, post, delete post...) and  it is also helpfull for debuging, as it gives you the the precision of the  exact feature causing this.

Reply:
# Sure 

Hi @andersonlebon 
I will take care of  it in my Free time To make my test and Website fully functional and Presentable 💕💕💪💪